### PR TITLE
Add SCURRENT_TIMESTAMP_MS for Millisecond Timestamps and SQLiteUtils::getRandomID Function and Use it in Jobs Plugin

### PR DESCRIPTION
**Pull request by expensifyrandomdrake**
This adds a new SCURRENT_TIMESTAMP_MS function to get a timestamp with milliseconds attached to it like: `2020-11-04 22:13:48.276`

This also adds a new `SQLiteUtils` file with `getRandomID` in it.

This also replaces the `getNextID` function in the `Jobs` plugin with usage of the new function.

### CODE REVIEW

File: sqlitecluster/SQLiteUtils.h [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+class SQLiteUtils {
+  public:
+      /**
+       * Gets the next ID available for random IDs on a given table and column
```
Comment (tylerkaraszewski) [inline sqlitecluster/SQLiteUtils.h:6]: This comment doesn't make much sense with "next ID available"

File: sqlitecluster/SQLiteUtils.h [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+       *
+       * @param db
+       * @param tableName
+       * @param column
+       * @return the new ID
```
Comment (tylerkaraszewski) [inline sqlitecluster/SQLiteUtils.h:11]: Also we stopped doing the javadoc style param/return stuff for C++ where the code itself does the exact same thing.

**Review by tylerkaraszewski — CHANGES_REQUESTED** [review]

**Comment by expensifyrandomdrake** [comment]
Updated based on feedback from review.

File: libstuff/libstuff.h [hunk 3]
```diff
@@ -712,6 +712,14 @@ inline string SUNQUOTED_TIMESTAMP(uint64_t when) { return SComposeTime("%Y-%m-%d
 inline string STIMESTAMP(uint64_t when) { return SQ(SUNQUOTED_TIMESTAMP(when)); }
 inline string SUNQUOTED_CURRENT_TIMESTAMP() { return SUNQUOTED_TIMESTAMP(STimeNow()); }
 inline string SCURRENT_TIMESTAMP() { return STIMESTAMP(STimeNow()); }
+inline string SCURRENT_TIMESTAMP_MS() {
```
Comment (tylerkaraszewski) [inline libstuff/libstuff.h:715]: I should have done this in the previous review, but I'm also gonna ask you to make this not `inline` and move the implementation into libstuff.cpp

Comment (expensifyrandomdrake) [reply libstuff/libstuff.h:715]: Would `STime.cpp` be a more appropriate place to put this?

Comment (tylerkaraszewski) [reply libstuff/libstuff.h:715]: It could be. I don't have strong feelings on it.

File: libstuff/STime.cpp [hunk 4]
```diff
@@ -50,3 +50,13 @@ timeval SToTimeval(uint64_t when) {
     // Just split by high and low bits
     return {(time_t)(when / STIME_US_PER_S), (suseconds_t)(when % STIME_US_PER_S)};
 }
+
+// --------------------------------------------------------------------------
```
Comment (tylerkaraszewski) [inline libstuff/STime.cpp:54]: Lol, annoying nitpick comments but we don't need this divider line thingy.

**Review by tylerkaraszewski — APPROVED** [review]

**Comment by tylerkaraszewski** [comment]
Looks good. Waiting for travis.

**Pull request merged**